bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Region.java

```diff
@@ -616,7 +623,204 @@ public void translate (Point pt) {
 @Override
 public String toString () {
 	if (isDisposed()) return "Region {*DISPOSED*}";
-	return "Region {" + handle + "}";
+	return "Region {" + zoomToHandle.entrySet().stream().map(entry -> entry.getValue() + "(zoom:" + entry.getKey() + ")").collect(Collectors.joining(","));
 }
 
+@FunctionalInterface
+private interface OperationStrategy {
+	void apply(Operation operation, long handle, int zoom);
+}
+
+abstract class Operation {
+	OperationStrategy operationStrategy;
```

Comment:
Better have this field `private` and add a constructor setting the field, which is then used as the super constructor for all subclasses.

Reply:
Done